### PR TITLE
(fx-#025): working on quality assurance of the project

### PR DESCRIPTION
**Pull request by kunda4**
# PR Description

This Pull Request is about working on Quality Assurance of our Website cloning

## Description of tasks that were expected to be completed

List all the tasks and sub-tasks that were assigned to you Make sure you check every completed task. list them in this style

- [x] changing the font weight for the text links
- [x] Adding the required spacing between the links
- [x] increasing the font size for the texts
- [x] Increasing the image size and changing the font style
- [x] Adjusting the padding on x-axis
- [x] Adjusting the background color

## Description of tasks that were expected to be completed and that are not yet completed

- [x] none

## How this may be tested?

Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

1. run `npm i` to install all required dependencies
2. run  `npm run build`
3. run `npm run start`
4. step 3
5. step 4...

## Screenshots (If appropriate/applicable)

Provide screenshots related to your PR in case it's appropriate and applicable

## Please check this Checklist before you submit your PR:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My code generates no warnings
- [x] There are no vulnerabilities

### CODE REVIEW

**Review by devfabien — APPROVED** [review]
looking good

File: src/components/data.js [hunk 1]
```diff
@@ -73,8 +73,8 @@ const skillSet = () => {
     .map(
       (
         elem
-      ) => `<div class=" flex text-textColorwhite md:gap-3 gap-1 font-para text-sm"> <h2 class="font-semibold md:py-2"> ${elem.name}
-        <p class="italic font-extralight py-2 text-xs">${elem.description}</p>
+      ) => `<div class=" flex text-textColorwhite md:gap-3 gap-1 font-para text-sm"> <h2 class="font-bold md:py-2"> ${elem.name}
+        <p class="italic font-extralight py-4 text-lg">${elem.description}</p>
```
Comment (kareraolivier) [inline src/components/data.js:106]: you are using the large font size

File: src/components/data.js [hunk 2]
```diff
@@ -91,7 +91,7 @@ const skillSet = () => {
   myDiv.innerHTML = `${datas
     .map(
       (skill) =>
-        `<div class="rounded-2xl py-1 px-2 h-fit bg-white whitespace-nowrap text-backgroundColor">${skill} </div>`
+        `<div class="rounded-2xl py-1 px-2 h-fit bg-white whitespace-nowrap text-backgroundColor font-title text-sm">${skill} </div>`
```
Comment (kareraolivier) [inline src/components/data.js]: add the font weight

File: src/components/data.js [hunk 3]
```diff
@@ -111,7 +111,7 @@ const skillSet = () => {
   myDiv2.innerHTML = `${datas2
     .map(
       (skill) =>
-        `<div class="rounded-2xl py-1 px-2 h-fit bg-white whitespace-nowrap text-backgroundColor">${skill} </div>`
+        `<div class="rounded-2xl py-1 px-2 h-fit bg-white whitespace-nowrap text-backgroundColor font-title text-sm">${skill} </div>`
```
Comment (kareraolivier) [inline src/components/data.js]: add the font weight of medium

File: src/sections/trainingProgram.html [hunk 4]
```diff
@@ -7,7 +7,7 @@
         <div class="bg-backgroundColor w-full lg:px-20 px-4">
             <div class="py-8  lg:w-1/2 flex flex-col gap-10">
                 <h3 class="font-title text-lg font-bold text-textColorwhite">Qualität steht bei uns an erster Stelle.
-                    Mit unserem <span class="text-primary">
+                    Mit unserem <span class="text-primary/50">
```
Comment (kareraolivier) [inline src/sections/trainingProgram.html]: remove the opacity on the text color

**Review by kareraolivier — CHANGES_REQUESTED** [review]
most of the things are good  but i have pointed out some changes

**Review by kareraolivier — COMMENTED** [review]
Conglats

**Review by kareraolivier — APPROVED** [review]
Ok

**Pull request merged**